src/cli.ts

```diff
@@ -66,6 +67,12 @@ export const optDefs: commandLineUsage.OptionDefinition[] = [
     type: (flag: string) => flag.split(',').map(Number),
     defaultValue: [8080, 8081, 8082, 8083, 0],
   },
+  {
+    name: 'config',
+    description: 'Path to JSON config file (see README for format)',
```

Comment:
We should document how this flag relates to others, or maybe error out if they specify a config and also opts that would be ignored given that there's a config.

Reply:
I made it error out for the flags that were previously just ignored. Also a bit of refactoring, since it's no longer possible to use the command-line parser's built-in default value handling :(